doc/helpers/Input-extensions.md

```diff
@@ -0,0 +1,27 @@
+# InputExtensions Attached Properties
+Provides various attached properties for _input controls_, such as `TextBox` and `PasswordBox`.
+
+## Properties
+Property|Type|Description
+-|-|-
+AutoDismiss|bool|Whether the soft-keyboard will be dismissed when the enter key is pressed.
+AutoFocusNext|bool|Whether the focus will move to the next focusable element when the enter key is pressed.\*
+AutoFocusNextElement|Control|Sets the next control to focus when the enter key is pressed.\*
```

Comment:
I think it would be insightful to write why you offer both the bool and the element.

I'm not sure we need the bool, seems it adds to the contract for low value.

Reply:
Maybe I couldve explained better..?:

So you have the option to move the focus to ... on enter:
- the next focusable control: `AutoFocusNext=True`
- a specified control: `AutoFocusNextElement={Binding ElementName=ThatOneOverThere}
The 2nd option is offered as a fallback solution when the 1st option doesnt behave as expected, or as an alternative when you need the focus to jump in a custom way

Reply:
> note: this is still in draft, as InputExtensions.EnterCommand will be aliased from ControlExtensions.Command and there is support for ListView item-clicks and navigation-controls navigation to be added

WE're not creating 2 public contracts, right? Will all be in ControlExtensions.Command, not in InputExtensions.EnterCommand

Reply:
@francoistanguay That was still yesterday's code. There is only `ControlExtensions.Command` now.